controllers/observations_c.js

```diff
@@ -95,7 +95,7 @@ observationController.addObservation = async function (req, res) {
     });
 
     return res.status(StatusCodes.OK).json(result);
-  } catch (error) {
```

Comment:
Don't we want to leverage the `error.message` here like in other controllers for a more informative reason of why the request failed?

Reply:
Of course it will need to be a separate commit if you decide to implement it, but perhaps I did not explain myself. I was asking why we were not leveraging the error message in the response -instead of returning a generic message- like we are doing in other controllers.